core/src/main/kotlin/gropius/model/architecture/InterfaceDefinition.kt

```diff
@@ -0,0 +1,91 @@
+package gropius.model.architecture
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.generator.annotations.GraphQLIgnore
+import gropius.model.common.ExtensibleNode
+import gropius.model.template.BaseTemplate
+import gropius.model.template.InterfaceDefinitionTemplate
+import gropius.model.template.MutableTemplatedNode
+import gropius.model.user.permission.ComponentPermission
+import gropius.model.user.permission.NodePermission
+import io.github.graphglue.model.*
+import org.springframework.data.annotation.Transient
+import org.springframework.data.neo4j.core.schema.CompositeProperty
+
+@DomainNode
+@GraphQLDescription(
+    """InterfaceSpecificationVersion on a ComponentVersion
```

Comment:
Is it possible that you renamed the class to InterfaceDefinition and forgot to update the doc?

And why did you update that ayway?

Reply:
no, InterfaceSpecificationVersion still exists  
however I copy pasted stuff and then adapted it, and forgot to change this 
fixed with 780b609a67a5caef014a483856ebb9eaa53732cb